packages/irmin-mirage-git/irmin-mirage-git.3.6.0/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+maintainer:   "thomas@gazagnaire.org"
+authors:      "Thomas Gazagnaire"
+license:      "ISC"
+homepage:     "https://github.com/mirage/irmin"
+bug-reports:  "https://github.com/mirage/irmin/issues"
+dev-repo:     "git+https://github.com/mirage/irmin.git"
+doc:          "https://mirage.github.io/irmin/"
+
+build: [
+ ["dune" "subst"] {dev}
+ ["dune" "build" "-p" name "-j" jobs]
+]
+
+depends: [
+  "dune"         {>= "2.9.0"}
+  "irmin-mirage" {= version}
+  "irmin-git"    {= version}
+  "mirage-kv"    {>= "3.0.0" & < "5.0.0"}
```

Comment:
is this upper-bound still required? mirage-kv was released 6 months ago

Reply:
Yeah, we haven't added support for the new functions added in 5.0.0. It's low priority but here is the issue https://github.com/mirage/irmin/issues/2083